.augment/rules/reviewer-guide.md

```diff
@@ -0,0 +1,301 @@
+- [ ] Query result is aliased to `result:`
+- [ ] Query is in a `.graphql` file in the appropriate `queries/` subdirectory
+- [ ] Generated `queries.go` file has been updated (run `go generate ./...`)
+- [ ] No manual edits to generated `queries.go` files
+
+**Common Issues**:
+- Custom query names instead of `RubrikPolarisSDKRequest`
+- Missing `result:` alias
+- Manually editing generated files
+- Forgetting to run `go generate ./...`
+
+**Example Feedback**:
+```
+❌ Wrong query name:
+query GetAwsAccounts($id: String!) {
+  allAwsCloudAccounts(id: $id) { ... }
+}
+
+✅ Should be:
+query RubrikPolarisSDKRequest($id: String!) {
+  result: allAwsCloudAccounts(id: $id) { ... }
+}
+
+Note: After adding/modifying .graphql files, run `go generate ./...` from the repository root.
+```
+
+### 3. Region Type Enforcement
+
+#### Type Safety
+- [ ] Cloud regions use typed region types (`aws.Region`, `azure.Region`, `gcp.Region`)
+- [ ] No string types used for region parameters
+- [ ] No string types used for region struct fields
+- [ ] Region parsing uses appropriate `RegionFrom*()` functions
+- [ ] Region conversion to GraphQL enums uses `.ToRegionEnum()` or similar methods
+- [ ] Invalid regions are handled (check for `RegionUnknown`)
+
+**Common Issues**:
+- Using `string` instead of region types
+- Not validating region input
+- Not handling `RegionUnknown` case
+
+**Example Feedback**:
+```
+❌ Using string for region:
+func AddExocompute(ctx context.Context, region string) error { ... }
+
+✅ Should use region type:
+func AddExocompute(ctx context.Context, region aws.Region) error {
+    if region == aws.RegionUnknown {
+        return fmt.Errorf("invalid region")
+    }
+    regionEnum := region.ToRegionEnum()
+    // ...
+}
+```
+
+### 4. Code Quality
+
+#### Error Handling
+- [ ] All errors are handled explicitly (no ignored errors)
+- [ ] Errors are wrapped with context using `fmt.Errorf("...: %w", err)`
+- [ ] Error messages are descriptive and helpful
+- [ ] Error messages start with lowercase (Go convention)
+
+**Example Feedback**:
+```
+❌ Ignored error:
+data, _ := json.Marshal(obj)
+
+✅ Should handle error:
+data, err := json.Marshal(obj)
+if err != nil {
+    return fmt.Errorf("failed to marshal object: %w", err)
+}
+```
+
+#### Function Design
+- [ ] Functions are focused and single-purpose
+- [ ] Functions use `context.Context` for cancellation and timeouts
+- [ ] Functions have reasonable complexity (not too long or nested)
+- [ ] Repeated code is extracted into helper functions
+
+#### Consistency
+- [ ] Code follows existing patterns in the codebase
+- [ ] Similar functionality uses similar approaches
+- [ ] Naming is consistent with related code
+
+### 5. Testing
+
+#### Test Coverage
+- [ ] New functionality has appropriate tests
+- [ ] Tests follow existing patterns in the codebase
+- [ ] Tests use meaningful names (e.g., `TestFunctionName_Scenario`)
+- [ ] Tests cover error cases
+- [ ] Tests are deterministic (no flaky tests)
```

Comment:
Maybe mention that table driven tests are preferred if they can be implemented without too much complexity.